dev-lang/ruby/ruby-2.3.1.recipe

```diff
@@ -0,0 +1,107 @@
+SUMMARY="A programming language focused on simplicity and productivity"
+DESCRIPTION="Ruby is a dynamic, reflective, object-oriented, general-purpose \
+programming language. It was designed and developed in the mid-1990s \
+by Yukihiro 'Matz' Matsumoto in Japan.
+
+Ruby embodies syntax inspired by Perl with Smalltalk-like features and \
+was also influenced by Eiffel and Lisp.  It supports multiple \
+programming paradigms, including functional, object oriented, and \
+imperative. It also has a dynamic type system and automatic memory \
+management. Therefore, it is similar in varying degrees to, Smalltalk, \
+Python, Perl, Lisp, Dylan, and CLU."
+HOMEPAGE="http://www.ruby-lang.org"
+COPYRIGHT="1993-2015 Yukihiro Matsumoto"
+LICENSE="Ruby
+	BSD (3-clause)"
+REVISION="1"
+SOURCE_URI="http://cache.ruby-lang.org/pub/ruby/2.3/ruby-$portVersion.tar.gz"
+CHECKSUM_SHA256="b87c738cb2032bf4920fef8e3864dc5cf8eae9d89d8d523ce0236945c5797dcd"
+PATCHES="ruby-$portVersion.patchset"
+
+ARCHITECTURES="!x86_gcc2 x86 x86_64"
+SECONDARY_ARCHITECTURES="x86"
+
+PROVIDES="
+	ruby$secondaryArchSuffix = $portVersion compat >= 2.2
+	cmd:ruby = $portVersion compat >= 2.2
+	cmd:erb = $portVersion compat >= 2.2
+	cmd:gem = $portVersion compat >= 2.2
+	cmd:irb = $portVersion compat >= 2.2
+	cmd:rake = $portVersion compat >= 2.2
+	cmd:rdoc = $portVersion compat >= 2.2
+	cmd:ri = $portVersion compat >= 2.2
+	cmd:testrb = $portVersion compat >= 2.2
+	lib:libruby$secondaryArchSuffix = $portVersion compat >= 2.2
+	"
+REQUIRES="
+	haiku$secondaryArchSuffix
+	lib:libedit$secondaryArchSuffix
+	lib:libz$secondaryArchSuffix
+	lib:libcrypto$secondaryArchSuffix
+	lib:libssl$secondaryArchSuffix
+	lib:libiconv$secondaryArchSuffix
+	lib:libncursesw$secondaryArchSuffix
+	"
+
+PROVIDES_devel="
+	ruby${secondaryArchSuffix}_devel = $portVersion
+	devel:libruby_static$secondaryArchSuffix = $portVersion compat >= 2.2
+	lib:libruby_static$secondaryArchSuffix = $portVersion compat >= 2.2
```

Comment:
Shouldn't **`lib:libruby_static`** go to PROVDES (instead of PROVIDES_devel)?
In other words, if any other recipe ever needs it, would it be desirable to make it depend on ruby_devel?

Does haikuporter complain if you build in strict mode? (**`haikuporter -S ruby-2.3.1`**)